backend/volontulo_org/settings/base.py

```diff
@@ -143,3 +143,5 @@
 LOGIN_URL = 'login'
 
 SYSTEM_DOMAIN = 'localhost'
+
+SENTRY_DSN = os.environ.get('VOLONTULO_SENTRY_DSN')
```

Comment:
Le'ts use `RAVEN_CONFIG` as the documentation suggest for loading Raven config.

Reply:
Done (though global `SENTRY_DSN` works properly). :P